services/crypto_update.proto

```diff
@@ -32,131 +39,195 @@ import "timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
 /**
- * Change properties for the given account. Any null field is ignored (left unchanged). This
- * transaction must be signed by the existing key for this account. If the transaction is changing
- * the key field, then the transaction must be signed by both the old key (from before the change)
- * and the new key. The old key must sign for security. The new key must sign as a safeguard to
- * avoid accidentally changing to an invalid key, and then having no way to recover. 
- * If the update transaction sets the <tt>auto_renew_account</tt> field to anything other 
- * than the sentinel <tt>0.0.0</tt>, the key of the referenced account must sign.
+ * Modify the current state of an account.
+ *
+ * ### Requirements
+ * - The `key` for this account MUST sign all account update transactions.
+ * - If the `key` field is set for this transaction, then _both_ the current
+ *   `key` and the new `key` MUST sign this transaction, for security and to
+ *   prevent setting the `key` field to an invalid value.
+ * - If the `auto_renew_account` field is set for this transaction, the account
+ *   identified in that field MUST sign this transaction.
+ * - Fields set to non-default values in this transaction SHALL be updated on
+ *   success. Fields not set to non-default values SHALL NOT be
+ *   updated on success.
+ * - All fields that may be modified in this transaction SHALL have a
+ *   default value of unset (a.k.a. `null`).
+ *
+ * ### Block Stream Effects
+ * None
  */
 message CryptoUpdateTransactionBody {
+    // NOTE: Seven deprecated fields should probably be removed and the
+    //       field names reserved.
+    // reserved 4,5,6,7,10,11,12
```

Comment:
In some other file I saw we mention "reserved 4,5" as last line of the file. We could follow the same here ?

Reply:
Any `reserved` fields would typically be at the top of a message, so it is easy to find.
This comment is here because the fields are deprecated and there might be code changes needed to actually remove them and mark the field numbers and names reserved.